apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/r4/providers/pac/ClaimE2E.java

```diff
@@ -130,11 +276,40 @@ public void testClaimFindByPatientWithNoResultsAndPaginationRequestedExpect200()
     given()
         .spec(requestAuth)
         .expect()
-        .log()
-        .ifError()
```

Comment:
Why remove the error logging?  Seems like it would be helpful if the test fails for some reason to see what the response contained.

Reply:
I noticed these logs only print when running locally, since the log output is swallowed when running via command line, so it wasnt super useful. Also RestAssured considers an "error" only a non-200 status, which limits its usefulness further. I've been going back and forth on whether adding the log.ifError is worth the couple lines for the limited utility, when realistically if something fails (probably during development) someone can just temporarily add the .log.body() and see the issue.

Reply:
Logs from tests are available when running in IDEA though.  Looked like these tests were expecting 200 so logging on other status codes seems reasonable to me.